docs/typescript/quickstart/node-fetch.md

```diff
@@ -0,0 +1,120 @@
+---
+id: node-fetch
+title: Sample Application with the typescript SDK
+sidebar_label: Node-fetch sample
+tags:
+  - node-fetch
+  - typescript
+  - sdk
+  - tutorial
+---
+
+#  Sample Node-fetch App 
+
+A sample node-fetch app containing GET and POST request integrated with keploy's [typescript-sdk](https://github.com/keploy/typescript-sdk)
+
+## Installation
+
+### Setup node-fetch app
+
+```bash
+git clone https://github.com/keploy/samples-typescript && cd samples-typescript/node-fetch
+yarn
+```
+
+### Start keploy server
+
+Run these commands for running kepoy server binary in MacOS.
+
+```bash
+curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_darwin_all.tar.gz" | tar xz -C /tmp
+
+sudo mv /tmp/keploy /usr/local/bin && keploy
+```
+
+For other operating system, follow [this guide](https://github.com/keploy/keploy#quick-installation)
+
+### Run the application
+
+```bash
+  source .env && node server.js
+```
+
+## Generate testcases
+
+To genereate testcases we just need to make some API calls. You can use [Postman](https://www.postman.com/), [Hoppscotch](https://hoppscotch.io/), or simply `curl`
+
+### 1. Post the data
+
+```bash
+curl --request POST \
+  --url http://localhost:8080/postData \
+  --header 'content-type: application/json' \
+  --data '{
+    "name": "Joey",
+    "job": "Actor"
+}'
+```
+this will return :
+```
+{
+  "id": "862",
+  "createdAt": "2022-10-07T09:43:40.211Z"
+}
+```
+
+### 2. Get the data
+```bash
+curl --request GET \
+  --url http://localhost:8080/getData
+```
+
+or by querying through the browser `http://localhost:8080/getData`
+
+
+Now both these API calls were captured as a testcase and should be visible in "keploy-tests" folder in project directory.
+If you're using Keploy cloud, open [this](https://app.keploy.io/testlist).
+
+You should be seeing ` readable/editable yaml files` with their mocks in seperate yamls in "/mock" folder.
+
+
+Now, let's see the magic! 🪄💫
+
```

Comment:
Can you include some images of keploy server when it shows the recorded test cases and mocks when API calls are made?